models/models.users.js

```diff
@@ -89,3 +89,22 @@ exports.removeUser = async (user_id) => {
 
   await User.remove({ _id: user_id})
 }
+
+
+exports.createOrder = async (user_id, order) => {
+  const expectedKeys = ['services', 'createdAt', 'fulfilledAt', 'servicedBy']
+  const receivedKeys = Object.keys(order)
+  const user = await User.findById(user_id);
```

Comment:
If we check for 404s after the `findOneAndUpdate` we can remove this `findById` call